src/oscillator.js

```diff
@@ -309,6 +321,13 @@ define(function (require) {
   p5.Oscillator.prototype.setType = function(type) {
     this.oscillator.type = type;
   };
+  /**
+   *  Returns  current type of oscillator eg. 'sine', 'triangle', 'sawtooth' or 'square'.
+   *
+   *  @method  getType
+   *  @for p5.Oscillator
+   *  @param {String} type 'sine', 'triangle', 'sawtooth' or 'square'.
```

Comment:
I think you meant `@returns`? This method doesn't accept a `@param` but those are the possible return values